### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by BarryHuffman**
Feel free to modify or exclude any of the content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# OpenWrt UBI installer image genarator
-for the Linksys E8450 aka. Belkin RT3200
+# An OpenWrt UBI installer image generator
+Supporting the: "Linksys E8450", and the "Belkin RT3200".(aka "Belkin AX3200")
 
 ![animated gif showing web UI and serial during installation](https://user-images.githubusercontent.com/9948313/108781223-78915500-7561-11eb-851a-3c4c744ad6c2.gif)
-(serial is RX only for documentation, only interaction is HTTP file upload)
+(The serial interface is displaying RX only for documentation purposes as the only interaction required is from within the web browser via HTTP file upload.
```
Comment (dangowrt) [inline README.md]: Maybe remove both parentheses?

File: README.md [hunk 2]
```diff
@@ -24,9 +26,30 @@ You may of course as well go ahead and download the generated files [here](https
...
+11. Upload the `openwrt-mediatek-mt7622-linksys_e8450-ubi-sysupgrade.itb` file.
+12. Reboot and proceed to a normal OpenWRT setup (or upload your configuration file).
+
+## Convert "release v0.3" to OpenWRT snapshot release.
+**WARNING** SNAPSHOT RELEASES ARE LARGELY UNTESTED! 
+THIS WILL ALSO REMOVE LUCI PACKAGES RESULTING IN A "HEADLESS" ROUTER WITH STOCK SETTINGS!
+PROCEED AT YOUR OWN RISK!
+1. If you haven't already, backup every "mtdblock" category, and move the "boot_backup" to another device. (In the event of emergency you can reflash via [JTAG](https://openwrt.org/toh/linksys/linksys_e8450)
+2. Backup your configuration file which can be loaded after.
+3. Upload the latest UBI [sysupgrade.itb](https://downloads.openwrt.org/snapshots/targets/mediatek/mt7622/openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb)
```
Comment (dangowrt) [inline README.md:43]: I'd recommend people to install `auc`, in that way all configuration and package selection is preserved.
```
opkg install auc
auc
```

**Review by dangowrt — APPROVED** [review]
Thank you very much for your contribution!
If you like to give `auc` or `luci-app-attendedsysupgrade` a try, as that allows easier upgrade to snapshot (as well as subsequent updates)

**Comment by BarryHuffman** [comment]
Thank you for the input I've refactored the process to include your suggestions.

**Comment by BarryHuffman** [comment]
I realize I misunderstood you, as you meant using acu prior as the method to snapshot release?
Could you modify that section to clarify as I've not tried the method myself.

**Pull request merged**